include/GL/glui.h

```diff
@@ -442,7 +444,7 @@ class GLUIAPI GLUI_StdBitmaps
 {
 public:
     GLUI_StdBitmaps(); 
-    ~GLUI_StdBitmaps();
+    ~GLUI_StdBitmaps() = default;
 
```

Comment:
I don't have a preference either way, but if a user requested to remove the C++11 parts I would comply with them. So I would just say wait and see until a real world situation presents itself. 

Just don't use C++11 stuff without reason to. It's more work to compile. And certain features like intializer-lists make me extremely uncomfortable because they've only barely been implemented for a year or so on major compilers.

A library has to be accommodating, and that means that it usually only does C++11 up to the point that it's enabling the user to use C++11 with its feature set insider their application. The goal is not to use C++11 internally, but not to restrict the user's ability to do so.

A UI library is unusual somewhat because we normally think of UI as something that only the application touches on, so it lives much closer to the user than most library code does. (Though there's always the exception to the rule.)